_posts/2015-06-12-topics.md

```diff
@@ -871,3 +878,238 @@ This topic is deprecated. Please use [shipment:packed](#shipment_packed) instead
+                        "expiration_date" : "2021-10-05",
+                        "origination_date" : "2019-01-23",
+                        "quantity" :  "1.0000"
+                    }
+                ]
+            }
+        ],
+        "packages" : [{
+            "package_id" : 99,
+            "weight" : 10.2,
+            "weight_unit" : "lb",
+            "items" : [
+                {"order_item_id": "443313", "sku" : "sku123", "quantity" : 1}
+            ],
+            "package_data": [
+                {"order_item_id": "443313", "label": "Serial Number - 8 Characters", "value": "55285442"}
+            ],
+            "tracking_numbers" : [
+                "794657040105"
+            ],
+            "tracking_urls" : [
+                "https://www.fedex.com/apps/fedextrack/?action=track&tracknumbers=794657040105"
+            ]
+        }],
+        "webhook_increment_id" : 1,
+        "event_timestamp" : "2021-12-07T16:08:15+00:00"
+    }
+}
+```
+
+<h3 id="shipment_delivered">
+    shipment:delivered
+</h3>
+
+```json
+{
+    "topic" : "shipment:delivered",
+    "message" : {
+        "unique_id" : "100000007",
+        "warehouse_id": 1,
+        "status" : "delivered",
+        "order_status" : "complete",
+        "order_ref" : "9099",
+        "order_unique_id" : "100000018",
+        "source" : "shopify",
+        "carrier" : "fedex",
+        "service" : "FEDEX_2_DAY",
+        "service_description" : "FedEx 2 Day",
+        "items" : [
+            {
+                "order_item_id" : "443313",
+                "order_item_ref" : null,
+                "sku" : "sku123",
+                "quantity" : "1.0000",
+                "order_item_qty" : "1.0000",
+                "lot_data" : [
+                    {
+                        "lot_number" : "83CBC2",
+                        "expiration_date" : "2021-10-05",
+                        "origination_date" : "2019-01-23",
+                        "quantity" :  "1.0000"
+                    }
+                ]
+            }
+        ],
+        "packages" : [{
+            "package_id" : 99,
+            "weight" : 10.2,
+            "weight_unit" : "lb",
+            "items" : [
+                {"order_item_id": "443313", "sku" : "sku123", "quantity" : 1}
+            ],
+            "package_data": [
+                {"order_item_id": "443313", "label": "Serial Number - 8 Characters", "value": "55285442"}
+            ],
+            "tracking_numbers" : [
+                "794657040105"
+            ],
+            "tracking_urls" : [
+                "https://www.fedex.com/apps/fedextrack/?action=track&tracknumbers=794657040105"
+            ]
+        }],
+        "webhook_increment_id" : 1,
+        "event_timestamp" : "2021-12-07T16:08:15+00:00"
+    }
+}
+```
+
+<h3 id="order_shipped">
+    order:shipped
+</h3>
+
+```json
+{
+    "topic" : "order:shipped",
+    "message" : {
+        "unique_id" : "100000022",
```

Comment:
Add "order_ref" to all "order:*" topics. This is already present in the code, just noticed it is missing from the docs..